lib/utils/getInputs.js

```diff
@@ -0,0 +1,21 @@
+const core = require("@actions/core");
+
+const { getInput } = require("./getInput.js");
+const { isWindows } = require("./helpers.js");
```

Comment:
In what situation would this run on Windows?

Reply:
Good Question 💯 I personally can't think of any, however, I wanted to try and make this work across the three workflows supported runtimes: macOS, Windows & Linux. Someone may be doing something based on Windows development for AWS and need to run the lining and find it quicker/easier to make it run on Windows. (That's a guess). 

I just thought if we could get 100% coverage of OS, no one could come back and say "hey this version won't meet my use case", if that makes sense?

Reply:
Sure, makes sense 👍🏻 Might as well!